jenkins-design-language/src/js/components/TimeDuration.jsx

```diff
@@ -101,16 +116,19 @@ export class TimeDuration extends Component {
                     <span title={this.props.hint ? this.props.hint : '<1s'}>&#x3C;1s</span>
                 );
             }
+
             const {
                 locale = 'en',
-                displayFormat = 'd[d] h[h] m[m] s[s]',
-                liveFormat = 'd[d] h[h] m[m] s[s]',
-                hintFormat = 'M [months], d [days], h [hours], m [minutes], s [seconds]',
+                t,
             } = this.props;
+
+            const liveFormat = t('common.date.duration.format', { defaultValue: 'm[ minutes] s[ seconds]' });
+            const hintFormat = t('common.date.duration.hint.format', { defaultValue: 'M [month], d [days], h[h], m[m], s[s]' });
+
             moment.locale(locale);
             // in case we are in live update we are interested in seconds
             const duration = this.props.liveUpdate ?
-                moment.duration(millis).format(liveFormat) : moment.duration(millis).format(displayFormat);
+                moment.duration(millis).format(liveFormat) : TimeDuration.format(millis, t, locale);
```

Comment:
Pretty sure that moment.duration is no longer needed here as liveformat and display format at the same. At the first lesat your format function should be the one to deal with the live format so it takes in <1s into account.

Reply:
@imeredith are you talking about the different formats here?
<img width="397" alt="screen shot 2017-10-17 at 7 43 49 am" src="https://user-images.githubusercontent.com/50156/31634373-fb619a4e-b30e-11e7-8d5b-34d37d6784d4.png">